wgsl/index.bs

```diff
@@ -10785,8 +10785,6 @@ The most complex rule is for function calls:
 
 Note: Refer to [[#func-var-value-analysis]] for the definition of *Vout*(*call*).
 
-A [=value constructor=] expression is analyzed as if it were a function call with the same arguments.
```

Comment:
This is unrelated.

Reply:
Yes it is an editorial only change since we moved the value constructors to built-in functions. See the 2nd commit message.